### ISSUE TITLE
I am not able to reduce the font size of groups values.

### ISSUE THREAD

**Issue by frecloud** [endangeredoxen/fivecentplots#44]
I want to reduce the **fontsize** of overlapping values of **'RegWr00_DATA_PEDESTAL_'**
So that user can read the values clearly.
I don't want to increase the ax_size[0], 
Just want to reduce the font size of these value to make them readable.
I tried diff kwargs from doc but none of them isn't working.
Thanks in advance, please assist me.
![img](https://user-images.githubusercontent.com/104160748/183258756-24a60f3b-7f78-40e6-bcb1-e60b700df3ae.jpeg)

**Comment by frecloud** [comment]
Also I can't use` tick_labels_minor_x_rotation=45` or `tick_labels_major_x_font_color='#FF0000'`
why is that?

**Comment by endangeredoxen** [comment]
You can reduce the box label font sizes using `box_group_label_font_size`

```
import fivecentplots as fcp
from pathlib import Path
![custom_box_group_label_font_size](https://user-images.githubusercontent.com/14165793/183259616-01087a01-d878-4aef-a42d-b77f6c14b39a.png)

import pandas as pd
df = pd.read_csv(Path(fcp.__file__).parent / 'test_data/fake_data_box.csv')
fcp.boxplot(df, y='Value', groups=['Batch', 'Sample'], box_group_label_font_size=6)
```

**Comment by endangeredoxen** [comment]
I have no issue changing the tick label color

```
fcp.plot(df, x='Voltage', y='I [A]', legend=['Die', 'Substrate'], ax_size=[400, 300],
              filter='Target Wavelength==450 & Temperature [C]==25 & Boost Level==0.2', 
              tick_labels_major_x_font_color='#FF0000')
```

but I'm seeing some weirdness with the minor ticks and will get back to you

![tick_labels_font_color](https://user-images.githubusercontent.com/14165793/183259833-59fdd183-d958-44f2-910c-584898297507.png)

**Comment by endangeredoxen** [comment]
If you get different results than what I posted above, please try to install fcp in a clean env.  If it works there then you may have some issue in your current env

**Comment by frecloud** [comment]
It is working, thanks.
One more thing, do we have something for rotation also for box_group_..... ?

**Comment by frecloud** [comment]
Where I can find the docs for box_group_ kwargs ?

**Comment by endangeredoxen** [comment]
See the link I posted above.  All kwargs are detailed in the API section of the docs. 

Custom rotation of box labels is not currently supported.  The default rotation is 0 deg.  If the labels are too wide they will automatically rotate to 90 deg.  Do you need this feature?

**Comment by endangeredoxen** [comment]
To get back to your tick question, the following should work:
```
import fivecentplots as fcp
import pandas as pd
from pathlib import Path
df = pd.read_csv(Path(fcp.__file__).parent / 'test_data' / 'fake_data.csv')
fcp.plot(df, x='Voltage', y='I [A]', legend=['Die', 'Substrate'], ax_size=[400, 300],
               filter='Target Wavelength==450 & Temperature [C]==25 & Boost Level==0.2', 
               tick_labels_major_x_font_color='#FF0000', tick_labels_minor=True, tick_labels_minor_x_rotation=45)
```

and indeed it does work--sometimes
![tick_rotation_minor](https://user-images.githubusercontent.com/14165793/183314752-7aa19469-8b2d-4c7c-92b8-2baacea95307.png)

However, I can run the *exact* same block of code in the same python env and sometimes the ticks get mangled like this:
![ticks_bad](https://user-images.githubusercontent.com/14165793/183314780-c49807f7-ce00-4c6c-9b59-f92273bb5bac.png)

I don't know where this bug is coming from but will continue investigating

**Comment by endangeredoxen** [comment]
I have found and resolved this bug.  The fix will be included in the beta of v0.5.0.  Thanks
